### ISSUE TITLE
Modelling of COVID-19 vaccines will change

### ISSUE THREAD

**Issue by lubianat** [egonw/SARS-CoV-2-Queries#43]
Hello, 

I raised a point in a recent discussion post about the modelling of vaccines on Wikidata: https://www.wikidata.org/wiki/Wikidata_talk:WikiProject_COVID-19#Ontological_organization_of_vaccines

In short, I'm planning to model  the different types of vaccines as types of vaccine, and not "vaccines" themselves.
I think this might break the vaccines section. A small change (P31 to P279) in the SPARQL query would provide support. 

I'd happily open a pull request with the updated query. Should I?

**Comment by egonw** [comment]
Yes, please go ahead!

**Comment by lubianat** [comment]
I've tried opening a PR, is that format correct?

**Comment by egonw** [comment]
Yes, but wrong folder. Everything in `docs/` is generated. Look up the query in `src/sparql/` please. Sorry for the confusion.

**Comment by lubianat** [comment]
Oh no, I did it again, sorry. I'm gonna fix it.
